src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
```

Comment:
It is the TFS build - you will need to make the changes there as well so that we can generate symbol packages for Arm32 build. @dagood should be able to give pointers on how to do the same.